package.json

```diff
@@ -17,7 +17,6 @@
         "backbone": "npm:backbone#~1.3.2",
         "backbone.marionette": "4.1.3",
         "backbone.radio": "^2.0.0",
-        "blazy": "1.8.2",
```

Comment:
🎉

Reply:
love the smell of deleted dependencies in the morning...